tests/aem_hw.pm

```diff
@@ -364,18 +364,34 @@ sub clear_tpm_dasharo {
     # select TPM(2) Operation, position may depend on number of available PCR banks
     send_key_until_needlematch('dasharo_tpm2_operation', 'up', 10);
 
+    # The popup we're in now is very slow, it is fully reprinted after each
+    # move, and because of serial redirection it is clearly visible that each
+    # line is printed separately. Key presses are sometimes dropped if they
+    # happen too quickly during printing, and wait_screen_change may catch the
+    # change between the lines, before window gets printed in full. Adding a 1s
+    # delay between key presses seems to make navigation reliable.
+
     # 'TPM2_ClearControl + Clear' is 3 steps down
     send_key 'ret';
+    sleep 1;
+    send_key 'down';
+    sleep 1;
     send_key 'down';
+    sleep 1;
     send_key 'down';
-    send_key('down', wait_screen_change => 1);
+    sleep 1;
     if (match_has_tag('dasharo_tpm12_operation')) {
         # 'Force TPM Clear, Enable, and Activate' - 6 steps total, 3 left
         send_key 'down';
+        sleep 1;
+        send_key 'down';
+        sleep 1;
         send_key 'down';
-        send_key('down', wait_screen_change => 1);
+        sleep 1;
         save_screenshot;
-        send_key 'ret';   # returns directly back to device manager
+        # https://github.com/Dasharo/dasharo-issues/issues/1091
+        send_key 'ret';   # returns directly back to device manager - bug?
```

Comment:
Seems like a bug to me.

Reply:
PPI should work the same for TPM1.2 case when the bug is fixed, right?  Then it's worth mentioning that once the bug is addressed, device manager will need to be exited explicitly like in TPM2 case and F12 below will need to be pressed for TPM1.2 too, so whoever hits this change of behavior knows how to deal with it.

Reply:
Added some comments in https://github.com/3mdeb/openqa-tests-qubesos/compare/8b8ac925387b4eb75423770b214807c186f6f866..0332ab4ad85bd6bffc4970ed59f2fdb99c9d0447

Reply:
https://github.com/3mdeb/openqa-tests-qubesos/compare/0332ab4ad85bd6bffc4970ed59f2fdb99c9d0447..0582e03fea3b31f6cb96d067ef841606d8ba18d6#diff-5bcc1c5dc564aac714d0933cac3e9cb6d906077998db688ae82132d224af29acR393